code/models/OutputDataModel.py

```diff
@@ -0,0 +1,88 @@
+import logging as LOG
+
+from models.CoreMetrics import CoreMetrics
+from models.ReadabilityMetrics import ReadabilityMetrics
+
+
+class OutputDataModel:
+    def __init__(self):
+        LOG.debug(f"{__name__} init")
+        self.coreMetrics = CoreMetrics()
+        self.readabilityMetrics = ReadabilityMetrics()
+        self.category = ""
+        self.description = ""
+
+    def setCoreMetrics(self, coreMetrics: CoreMetrics) -> None:
```

Comment:
shouldn't we be following snake casing?

Reply:
Discussed and agreed to go ahead with camelCase